src/harness/unittests/matchFiles.ts

```diff
@@ -109,23 +109,21 @@ namespace ts {
         }
         {
             const actual = ts.parseJsonConfigFileContent(json, host, basePath, existingOptions, configFileName, resolutionStack);
-            expected.errors = map(expected.errors, error => {
-                return <Diagnostic>{
-                    category: error.category,
-                    code: error.code,
-                    file: undefined,
-                    length: undefined,
-                    messageText: error.messageText,
-                    start: undefined,
-                };
-            });
+            expected.errors = expected.errors.map<Diagnostic>(error => ({

```

Comment:
I've always felt specifying the return type on the lambda looked clunky :). It's too bad we don't use contextual typing here, since `expected.errors` has a definite type.